Sources/ShortcutFoundation/Core/Network/Core/Calls/NetworkingClient+NetworkingJSONDecodable.swift

```diff
@@ -13,7 +13,7 @@ public typealias NetworkPublisher<Response> = AnyPublisher<Response, NetworkingE
 
 public extension NetworkingClient {
 
-    func get<ResponsePayload: Decodable, Payload: Encodable>(_ route: String, params: Payload) -> NetworkPublisher<ResponsePayload> {
+    func get<ResponsePayload: Decodable, Payload: Encodable, errorPeyload: Encodable>(_ route: String, params: Payload, error: errorPeyload) -> NetworkPublisher<ResponsePayload> {
```

Comment:
typo: `errorPeyload`

Reply:
Also I do not want to send this in to all my requests. so you will have to create an overload.